### PR TITLE
IGAPP-1013: Add build config obdach

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: build-configs/obdach/assets/cityNotCooperatingTemplate.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const template =
```
Comment (steffenkleinle) [inline build-configs/obdach/assets/cityNotCooperatingTemplate.ts:1]: Delete

File: build-configs/obdach/assets/intro-slides/Events.svg [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (steffenkleinle) [inline build-configs/obdach/assets/intro-slides/Events.svg:1]: Delete all intro slides and native assets

File: build-configs/obdach/index.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  aboutUrls: {
+    default: 'https://vernetztesobdach.de/about/',
+    en: 'https://vernetztesobdach.de/en/about/'
+  },
+  privacyUrls: {
+    default: 'https://vernetztesobdach.de/datenschutz/',
+    en: 'https://vernetztesobdach.de/en/privacy/'
+  }
+}
+export const androidObdachBuildConfig: AndroidBuildConfigType = {
```
Comment (steffenkleinle) [inline build-configs/obdach/index.ts:45]: Did you try to make android/ios build configs optional?

File: build-configs/obdach/index.ts [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  ...commonObdachBuildConfig,
+  splashScreen: true,
+  applicationId: 'tuerantuer.app.integreat',
+  sha256CertFingerprint:
+    '66:2E:43:DC:9C:75:DE:7D:99:C5:BA:A9:19:DA:0F:BB:5E:6C:13:3D:03:E8:1D:FB:EF:87:F5:4B:F1:80:D4:B1',
+  googleServices: {
```
Comment (steffenkleinle) [inline build-configs/obdach/index.ts:51]: Remove google services (for iOS as well), only needed for push notifications on native

File: build-configs/obdach/index.ts [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+import { lightTheme } from './theme'
+
+const APPLE_APP_ID = '1072353915'
+
+const commonObdachBuildConfig: CommonBuildConfigType = {
+  appName: 'Vernetztes Obdach',
+  appIcon: 'app_icon_obdach',
+  lightTheme,
+  assets: OBDACH_ASSETS,
+  cmsUrl: 'https://admin.vernetztesobdach.de',
```
Comment (steffenkleinle) [inline build-configs/obdach/index.ts:19]: This is quite sure not the cms api url but the cms frontend. Should probably be `https://cms.vernetztesobdach.de`

File: build-configs/obdach/index.ts [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  hostName: 'vernetztesobdach.de',
+  allowedHostNames: ['admin.vernetztesobdach.de'],
```
Comment (steffenkleinle) [inline build-configs/obdach/index.ts:21]:
```suggestion
  allowedHostNames: ['cms.vernetztesobdach.de'],
```

File: build-configs/obdach/index.ts [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  internalLinksHijackPattern:
+    'https?:\\/\\/((admin\\.)?vernetztesobdach\\.de)(?!\\/(media|[^/]*\\/(wp-content|wp-admin|wp-json))\\/.*).*',
```
Comment (steffenkleinle) [inline build-configs/obdach/index.ts:23]:
```suggestion
    'https?:\\/\\/((cms\\.)?vernetztesobdach\\.de)(?!\\/(media|[^/]*\\/(wp-content|wp-admin|wp-json))\\/.*).*',
```

Comment (f1sh1918) [reply build-configs/obdach/index.ts:23]: already changed

File: build-configs/obdach/index.ts [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  featureFlags: {
+    floss: false,
+    pois: true,
+    newsStream: true,
+    pushNotifications: true,
+    introSlides: true,
```
[the remark this answers is not in the document]
Comment (steffenkleinle) [reply build-configs/obdach/index.ts]: It enables the whole news stream feature, i.e. the navigation tile with local and tünews

File: build-configs/obdach/index.ts (hunk 4 above)
Comment (steffenkleinle) [reply build-configs/obdach/index.ts:51]: i meant set it to `null`, you don't need to adjust the types for that.

**Review by steffenkleinle — COMMENTED** [review]
APPLE_APP_ID should be removed completely as it just points to the wrong app

**Comment by steffenkleinle** [comment]
> campaign logo and city not cooperating icon should be removed as well

still relevant, same for assets only used on native (if existent). Not that important though so I will approve to not block anything.

**Review by steffenkleinle — APPROVED** [review]

File: build-configs/obdach/assets/intro-slides/Events.svg (hunk 2 above)
Comment (f1sh1918) [reply build-configs/obdach/assets/intro-slides/Events.svg]: alright @steffenkleinle i removed unneeded assets

File: build-configs/obdach/index.ts (hunk 8 above)
[the remark this answers is not in the document]
Comment (f1sh1918) [reply build-configs/obdach/index.ts]: told with fritjof and it should be enabled @steffenkleinle

File: .circleci/config.yml [hunk 9]
```diff
@@ -1027,6 +1034,7 @@ workflows:
                     - build_ig_test_cms_web
                     - build_malte_test_cms_web
                     - build_aschaffenburg_web
```
Comment (steffenkleinle) [inline .circleci/config.yml:1036]: This won't work, you have to add it as a step to the `if development` in `deliver_web`

**Pull request merged**